sdk/compute/Azure.ResourceManager.Compute/src/autorest.md

```diff
@@ -82,15 +80,27 @@ directive:
   - rename-model:
       from: RestorePointCollection
       to: RestorePointGroup
-  - from: disk.json
-    where: $.definitions.PurchasePlan
-    transform: $["x-ms-client-name"] = "DiskPurchasePlan"
-  - from: swagger-document
-    where: $.definitions.VirtualMachineReimageParameters
-    transform: $["x-ms-client-name"] = "VirtualMachineReimageOptions"
-  - from: swagger-document
-    where: $.definitions.VirtualMachineScaleSetVMReimageParameters
-    transform: $["x-ms-client-name"] = "VirtualMachineScaleSetVmReimageOptions"
+  - rename-model:
```

Comment:
Consider to put `rename-model` at last because it changes the model name, then if one directiy after the rename which want to change something of the properties in this model, need to use the new name instead of the name in swagger spec. This will bring extra trouble if not aware the model name is renamed.

Reply:
Everything works here. I believe modeler will execute the rename-model directive at last